### PR TITLE
Translations update from Mattermost Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Mattermost Weblate](https://translate.mattermost.com) for [mattermost-languages-shipped/mattermost-server](https://translate.mattermost.com/projects/mattermost/mattermost-server_master/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: i18n/hr.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Willyfrog) [inline i18n/hr.json:1]: @ctlaltdieliet is this expected?

cc/ @m1lt0n

**Review by Willyfrog — APPROVED** [review]
looks ok, so I'm aproving, but raised a weird thing in one of the languages

File: i18n/hr.json (hunk 1 above)
Comment (m1lt0n) [reply i18n/hr.json:1]: In server it's not normal. @ctlaltdieliet this file was removed in the past right? the format for the server is that an array is expected not an object (even if there are no translations in it), so validation of translations fails on server.

Comment (ctlaltdieliet) [reply i18n/hr.json:1]: It was removed but Weblate or a user recreated it. I removed it once more.
Once we get the user permissions right in Weblate, user will no longer be able to create languages in shipped.
Only admins will be able to do that.

**Review by m1lt0n — APPROVED** [review]

File: i18n/hr.json (hunk 1 above)
Comment (m1lt0n) [reply i18n/hr.json:1]: I pulled the latest commits @ctlaltdieliet and hr.json made its way again into the repo :/

**Pull request merged**